src/detail-page/components/TabJobContent/JobTab.tsx

```diff
@@ -78,11 +84,12 @@ const JobTab: React.FC = () => {
       >
         Search
       </SearchButton>
-      {jobPosts.length > 0 ? null : <HiringImage src={hiringPicture} />}
       { isFetching ? <SearchSpinner><CircularProgress /></SearchSpinner> : null}
       <JobList>
         {jobPosts.map((job: Job) => <JobListItem job={job} key={job.id} />)}
+        <IsEmpty>{jobPosts.length > 0 ? null : <Typography variant="h2" color="textSecondary"> {emptySearchMessage} </Typography>}</IsEmpty>
```

Comment:
I don't think we need to get the empty search message from the hook. The message itself will never change; what we want is to manage it's visibility. 
We can easily achieve it like so:
```suggestion
      {!isFetching && !jobPosts.length && (
          <IsEmpty>
            <Typography variant="h2" color="textSecondary"> No job offers found. </Typography>
          </IsEmpty>
        )}
```
And then we can safely remove all logic related to emptySearchMessage in the hook :)
Finally, we could also extract the condition to a named var:
```
const isEmptySearchMessageVisible = !isFetching && !jobPosts.length;
```